### PR TITLE
Updated the Installation Guide fo RaspberryPi Raspbian/Ubuntu and Intel Up² Ubuntu

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Updated the Installation Guide fo RaspberryPi Raspbian/Ubuntu and Intel Up² Ubuntu

### CODE REVIEW

File: _posts/intro/2014-02-01-raspberry-pi-quick-start.md [hunk 1]
```diff
@@ -19,13 +19,11 @@ Raspberry Pi and the Kura development environment.
...
 
-<pre>2019-07-10-raspbian-buster-lite.img</pre>
+<pre>2021-10-30-raspios-bullseye-armhf.zip</pre>
 
 downloaded from
 
-<pre>https://www.raspberrypi.org/downloads/raspbian/</pre>
-
-and with the NOOBS image installed on the sd card bundled with the Raspberry Pi 3.
+<pre>https://www.raspberrypi.com/documentation/computers/remote-access.html</pre>
```
Comment (mstankovic) [inline _posts/intro/2014-02-01-raspberry-pi-quick-start.md]: this address points to documentation, maybe you can put link https://www.raspberrypi.com/software/operating-systems/

or direct link https://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2021-11-08/2021-10-30-raspios-bullseye-armhf.zip

File: _posts/intro/2014-02-01-raspberry-pi-quick-start.md [hunk 2]
```diff
@@ -38,58 +36,47 @@ in order to enable it to follow the instructions available at the following URL:
...
-
-4. Install the gdebi command line tool:
-
-    <pre>sudo apt-get update
-   sudo apt-get install gdebi-core</pre>
-
-5. Make sure that Java 8 is installed with
-
-    <pre>java -version</pre>
+1. Boot the Raspberry Pi with the latest Raspbian image (starting from release 5.1.0 Kura only supports Debian 11 or above).
```
Comment (MMaiero) [inline _posts/intro/2014-02-01-raspberry-pi-quick-start.md]: Are we sure that previous raspbian builds do not work? I believe we can say that 5.1.0 is tested with bullseye, right?

**Review by MMaiero — APPROVED** [review]

File: _posts/intro/2014-02-01-raspberry-pi-quick-start.md (hunk 2 above)
Comment (salvatore-coppola) [reply _posts/intro/2014-02-01-raspberry-pi-quick-start.md]: yeah, i'll modify.

File: _posts/intro/2014-02-01-raspberry-pi-quick-start.md (hunk 1 above)
Comment (salvatore-coppola) [reply _posts/intro/2014-02-01-raspberry-pi-quick-start.md]: i've updated

**Pull request merged**